core/interop/pinvoke/marshalling/readme.md

```diff
@@ -0,0 +1,39 @@
+.NET Core PInvoke Marshalling Sample
+================
+
+This project demonstrates different ways to marshall arguments to native function when using PInvokes. Documentation can be found here for [PInvokes](https://docs.microsoft.com/en-us/dotnet/standard/native-interop/pinvoke) and here for the [Type marshalling](https://docs.microsoft.com/en-us/dotnet/standard/native-interop/type-marshaling).
+
+Prerequisites
+------------
+
+* .NET Core 3.0 (at least Preview 6) - [https://dot.net](https://github.com/dotnet/core-sdk#installers-and-binaries)
+
+* C++ compiler
+  * Windows: `cl.exe`
+  * Linux/OSX: `g++`
+
+Build and Run
+-------------
+
+1) In order to build and run, all prerequisites must be installed. The following are also required:
+
+    * The C++ compiler (`cl.exe` or `g++`) must be on the path.
+      * On Windows, a [Developer Command Prompt for Visual Studio](https://docs.microsoft.com/cpp/build/building-on-the-command-line#developer_command_prompt_shortcuts) should be used.
+    * The C++ compiler (`cl.exe` or `g++`) and `dotnet` must be the same bitness.
```

Comment:
What is [`bitness` ](https://en.wiktionary.org/wiki/bitness) means?

Reply:
32bit versus 64bit in this context. I added some more explanation to make it clearer.